reference/7.4/Microsoft.PowerShell.Core/About/about_Arrays.md

```diff
@@ -646,7 +646,7 @@ The following example shows how to select all odd numbers from the array.
 This example show how to select the strings that aren't empty.
 
 ```powershell
-('hi', '', 'there').Where({$_.Length})
+('hi', '', 'there').Where{$_}
```

Comment:
If we're going to change this example at all, I would change it to use the **IsNullOrEmpty** static method. While you _can_ use the more concise `{ $_ }`, this can be more confusing for readers. Thinking on it, I would prefer the examples in this section to all return actual booleans, rather than rely on the behavior of casting to a boolean - it's a useful technique, but not what this article is documenting.

```suggestion
('hi', '', 'there').Where{ -not [string]::IsNullOrEmpty($_) }
```